src/runner.py

```diff
@@ -87,9 +87,10 @@ def _schedule_test(self, tarball_node, plan, device):
 class RunnerLoop(Runner):
     """Runner subclass to execute in a loop"""
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, configs, args, **kwargs):
+        super().__init__(configs, args, **kwargs)
         self._job_tmp_dirs = {}
+        self._plan = self._plan_configs[args.plan]
```

Comment:
`self._plan_configs[args.plan]` is used in `RunnerLoop` and `RunnerSingleJob` classes.
We can move it to `Runner` class.
```suggestion
```


The we can use it in `RunnerSingleJob.run()` too.
```
return self._run_single_job(tarball_node, self._plan,
                                    device_config)
```

Reply:
The difference is that `run` should be getting the test plan name from the command line argument, but `loop` should ideally be reading it from the YAML configuration like `kci_test`.  In fact it should get merged with `kci_test` eventually so we can run the new pipeline jobs with it too, but at the moment the list of template parameters is very different compared to LAVA jobs.  So in the meantime, there's a fixed test plan config stored in the object but it should be removed.  And for the `run` command, it shouldn't be stored in the object as it's specific to the call to the `run` method.